scopes/toolbox/regex/node-modules-exclude-packages/node-modules-exclude-packages.docs.md

```diff
@@ -0,0 +1,25 @@
+---
+labels: ['typescript', 'utils', 'packages', 'node modules']
+description: 'Exclude packages from node modules.'
+---
+
+import { nodeModulesExcludePackages } from './node-modules-exclude-packages';
+
+A function that receive an array of packages name to exclude from node modules and return a regex of it.  
+For example:
+
+```js live
+() => {
+  const packagesToTransform = [
+    'lit',
+    '@lit',
+    'testing-library__dom',
+    '@open-wc',
+    'lit-html',
+    'lit-element',
+    'pure-lit',
+    'lit-element-state-decoupler',
```

Comment:
don't need to have such long list in the example

Reply:
done